File: src/main/java/com/ega/bank/bank_management_system/dto/ClientDto.java
package com.ega.bank.bank_management_system.dto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ClientDto {
    private String nom;
    private String prenom;
    private Date birthday;
    private String sexe;
    private String telephone;
    private String email;
    private String address;
    private String nationalite;
}

File: frontEgaBank/src/app/core/models/client-dto.ts
export interface ClientDto {
  nom: string;
  prenom: string;
  birthday: Date;
  sexe: string;
  telephone: string;
  email: string;
  address: string;
  nationalite: string;
}

export interface UpdateClientDto {
  nom?: string;
  prenom?: string;
  birthday?: Date;
  sexe?: string;
  telephone?: string;
  email?: string;
  address?: string;
  nationalite?: string;
}
